frontend/packages/dev-console/src/components/topology/components/nodes/OperatorBackedService.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react';
+import * as classNames from 'classnames';
+import { connect } from 'react-redux';
+import {
+  Layer,
+  useHover,
+  Node,
+  createSvgIdUrl,
+  useDragNode,
+  observer,
+  useCombineRefs,
+  WithDndDropProps,
+} from '@console/topology';
+import { RootState } from '@console/internal/redux';
+import NodeShadows, { NODE_SHADOW_FILTER_ID_HOVER, NODE_SHADOW_FILTER_ID } from '../NodeShadows';
+import SvgBoxedText from '../../../svg/SvgBoxedText';
+import { getTopologyFilters, TopologyFilters } from '../../filters/filter-utils';
+import useFilter from '../../filters/useFilter';
+import { nodeDragSourceSpec } from '../../componentUtils';
+import { TYPE_OPERATOR_BACKED_SERVICE } from '../../const';
+import './OperatorBackedService.scss';
+
+export type OperatorBackedServiceProps = {
+  element: Node;
+  dragging?: boolean;
+  filters: TopologyFilters;
+  droppable?: boolean;
+  hover?: boolean;
+  highlight?: boolean;
+  regrouping: boolean;
+  canDrop?: boolean;
+  dropTarget?: boolean;
+} & WithDndDropProps;
```

Comment:
The factory is not adding `withDndDrop` or `withDragNode` so a lot of these props will not be passed.

Reply:
removed it.